uncorrected export

An error "TypeError: proxy_memoize__WEBPACK_IMPORTED_MODULE_6___default(...) is not a function" would pop out when I use this lib in a next.js project.

I also tied the solution like #22 

```js
import memoize from 'proxy-memoize/dist/index.modern';
```

but got "Module not found: Package path ./dist/index.modern is not exported from package"

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,6 +1,9 @@
 # Change Log
 
 ## [Unreleased]
+### Changed
+- CJS build for Node.js (#32)
+
 
 ## [0.3.6] - 2021-08-13
 ### Changed

--- a/package.json
+++ b/package.json
@@ -18,6 +18,7 @@
       "types": "./dist/src/index.d.ts",
       "module": "./dist/index.modern.js",
       "import": "./dist/index.modern.mjs",
+      "node": "./dist/wrapper.cjs",
       "default": "./dist/index.umd.js"
     }
   },
@@ -28,7 +29,7 @@
   ],
   "scripts": {
     "compile": "microbundle build -f modern,umd",
-    "postcompile": "cp dist/index.modern.mjs dist/index.modern.js && cp dist/index.modern.mjs.map dist/index.modern.js.map",
+    "postcompile": "cp dist/index.modern.mjs dist/index.modern.js && cp dist/index.modern.mjs.map dist/index.modern.js.map && cp src/wrapper.cjs dist/",
     "test": "run-s eslint tsc-test jest",
     "eslint": "eslint --ext .js,.ts --ignore-pattern dist .",
     "jest": "jest --preset ts-jest/presets/js-with-ts",

--- a/src/wrapper.cjs
+++ b/src/wrapper.cjs
@@ -1,0 +1,4 @@
+const { default: defaultFn, ...rest } = require('./index.umd.js');
+
+module.exports = defaultFn;
+Object.assign(module.exports, rest);

